.gitignore

```
@@ -7,12 +7,12 @@ package-lock.json
 yarn.lock
 pnpm-lock.yaml
 
+user/*
 
-plugins/*
 !plugins/.gitkeep
```

```suggestion
user/*
!user/*/.gitkeep
```
Also make a settings gitkeep while you're at it

---

@DracoClaw doesn't look like the gitkeep files were actually commited, can you double check?

---

should be there now